client/src/components/AppComponent.jsx

```diff
@@ -143,11 +152,8 @@ export default class AppComponent extends React.Component {
                 </div>
 
                 <Switch>
+
                     <ProtectedRoute
-                        path="/welcome"
-                        component={WelcomeComponent}
-                    />
-                    <Route
```

Comment:
wrote about it in corresponding Trello Task T 10.4: no sense of welcome page . brings nothing. instead of it I added the user's name to Navbar